tests/ui/pattern/usefulness/empty-types.min_exh_pats.stderr

```diff
@@ -403,18 +456,24 @@ error: unreachable pattern
    |
 LL |         _ => {}
    |         ^
+   |
+   = note: this pattern matches no values because `[!; 3]` is uninhabited
```

Comment:
We should look into making this the span label and maybe have the note be a link to an explanation of uninhabited types (!).